tests/testsuite/test.rs

```diff
@@ -3658,3 +3658,160 @@ fn test_dep_with_dev() {
         )
         .run();
 }
+
+#[cfg(target_env = "gnu")]
```

Comment:
Could this also be generalized to run on all platforms, perhaps doing something like `ignore-x86_64` and only running the test on 64-bit platforms? (which covers all our major OSes)